src/lib/tradesHandler.ts

```diff
@@ -227,7 +177,13 @@ export const performActions = (
   const messageArray: string[] = [];
   trades.forEach((trade) => {
     const { ticker, breakoutRef } = trade;
-    const newTradeStatus = determineTradeStatus(trade, stopLossLimit);
+    const buySellHelpers = getBuySellHelpers();
+    const newTradeStatus = buySellHelpers.determineTradeStatus({
```

Comment:
This caught my eye together with the `if (!newTradeStatus)` which was surprising. determineTradeStatus sounds like it would return the current trade status. newTradeStatus sounds like it would be the new status. Then it is used like a bool (thank you js), so I had to go look in the determineTradeStatus function to see what it returned.

I think this would be easier to read if it was more consistent. One suggestion is to make determineTradeStatus to always return the current trade status. Then you could do `if(newTradeStatus === trade.status)` below instead.

Reply:
fixed as suggested!